configs/defs/generic_linux.textproto

```diff
@@ -9558,3 +9558,649 @@ benchmark_configs: {
       "}"
   }
 }
+benchmark_configs: {
+  id: "ensure-password-failed-attempts-lockout-includes-root-account"
+  compliance_note: {
+    title: "Ensure password failed attempts lockout includes root account"
+    description: 
```

Comment:
Our internal linters require that there be no trailing whitespaces - e.g. this should be "description:", not "description: " (note the space at the end). Same for other lines below. IDEs like vs code have a setting to do this automatically.